include/matchit.h

```diff
@@ -104,13 +104,28 @@ namespace matchit
                            { return *id; });
         }
 
+        template <typename T>
+        constexpr auto expr(Nullary<T> const &v)
+        {
+            return v;
+        }
+
         template <typename T>
         constexpr auto expr(T const &v)
         {
             return nullary([&]
                            { return v; });
         }
 
+        template <typename T>
+        constexpr auto toNullary(T &&v)
+        {
+            if constexpr ( std::is_invocable_v<std::decay_t<T>> )
```

Comment:
Same here. Run single-header.sh should fix this as well.

Reply:
done